app/templates/components/forms/login-form.hbs

```diff
@@ -35,9 +35,13 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        <a href="{{href-to 'register'}}">
```

Comment:
This is incorrect semantic markup. `segment` should be a direct child of `segments`